Extensions/retags.js

```diff
@@ -91,6 +91,11 @@ XKit.extensions.retags = {
 
 	tag: function(elements) {
 		$(elements).each(function() {
+			if ($(this).hasClass("retags_has_processed")) {
```

Comment:
this needs to be after we allocate $element so we're not calling the jQuery constructor 3 times.....

Reply:
that's what i was trying to do by pausing the mutation observer... not sure how to call the function 3 times without the constructor being called 3 times too, if i'm understanding the term correctly. what am i missing? 
(i just noticed i didn't put `retags_has_processed` in `.reblog:not(...)` but that's still not as efficient as you're asking for, i think?)

Reply:
ah, sorry. these are *separate* "three times"